app/Http/Controllers/Entity/AbilityController.php

```diff
@@ -166,7 +166,7 @@ public function update(StoreEntityAbility $request, Entity $entity, EntityAbilit
     {
         $this->authorize('update', $entity->child);
 
-        $data = $request->only(['ability_id', 'visibility_id', 'note', 'position']);
+        $data = $request->only(['ability_id', 'visibility_id', 'note']);
```

Comment:
Unrelated, but the api endpoint no longer works on my local version
![image](https://user-images.githubusercontent.com/3263440/205393530-69426dc6-ea4c-4729-8b5a-381dcc9a12bf.png)